pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -99,6 +99,7 @@ type zedagentContext struct {
 	subEdgeNodeCert           pubsub.Subscription
 	subVaultStatus            pubsub.Subscription
 	subAttestQuote            pubsub.Subscription
+	subVaultKeyFromDevice     pubsub.Subscription
```

Comment:
The type is EncryptedVaultKey. Makes sense to have Encrypted in the name of the subscription and handlers.
Or if that is too long, drop the "Vault" part of the name?

Reply:
Corrected it.